OpenFAST/IEA-15-240-RWT/IEA-15-240-RWT_InflowFile.dat

```diff
@@ -1,52 +1,52 @@
 ================== Parameters for Uniform wind file   [used only for WindType = 2] ============================
-"Wind/NoShr_9-14_Inc1_50s.wnd" Filename_Uni    - Filename of time series data for uniform wind field.      (-)
-150.0                  RefHt_Uni       - Reference height for horizontal wind speed                (m)
-240.0                  RefLength   - Reference length for linear horizontal and vertical sheer (-)
+"none"                 Filename_Uni - Filename of time series data for uniform wind field.      (-)
+150.0                  RefHt_Uni   - Reference height for horizontal wind speed                (m)
+1.0                    RefLength   - Reference length for linear horizontal and vertical sheer (-)
 ================== Parameters for Binary TurbSim Full-Field files   [used only for WindType = 3] ==============
-"Wind/145m_11mps_ClassA1ETM.bts" FileName_BTS    - Name of the Full field wind file to use (.bts)
+"none"                 FileName_BTS - Name of the Full field wind file to use (.bts)
 ================== Parameters for Binary Bladed-style Full-Field files   [used only for WindType = 4] =========
-"Wind/90m_12mps_twr"   FilenameRoot - Rootname of the full-field wind file to use (.wnd, .sum)
+"none"                 FilenameRoot - Rootname of the full-field wind file to use (.wnd, .sum)
 False                  TowerFile   - Have tower file (.twr) (flag)
 ================== Parameters for HAWC-format binary files  [Only used with WindType = 5] =====================
-"wasp/Output/basic_5u.bin" FileName_u  - name of the file containing the u-component fluctuating wind (.bin)
-"wasp/Output/basic_5v.bin" FileName_v  - name of the file containing the v-component fluctuating wind (.bin)
-"wasp/Output/basic_5w.bin" FileName_w  - name of the file containing the w-component fluctuating wind (.bin)
-64                     nx          - number of grids in the x direction (in the 3 files above) (-)
-32                     ny          - number of grids in the y direction (in the 3 files above) (-)
-32                     nz          - number of grids in the z direction (in the 3 files above) (-)
-16.0                   dx          - distance (in meters) between points in the x direction    (m)
-3.0                    dy          - distance (in meters) between points in the y direction    (m)
-3.0                    dz          - distance (in meters) between points in the z direction    (m)
-150.0                  RefHt_Hawc  - reference height; the height (in meters) of the vertical center of the grid (m)
+"none"                 FileName_u  - name of the file containing the u-component fluctuating wind (.bin)
+"none"                 FileName_v  - name of the file containing the v-component fluctuating wind (.bin)
+"none"                 FileName_w  - name of the file containing the w-component fluctuating wind (.bin)
+2                      nx          - number of grids in the x direction (in the 3 files above) (-)
```

Comment:
Are these grid parameter changes correct?

Reply:
I don't think anyone has never used them... still I reverted to the old lines